openapi-cli/src/test/resources/generators/client/ballerina/path_parameter_with_special_name.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/http;
+
+# + clientEp - Connector http endpoint
+public client class Client {
+    http:Client clientEp;
+    # Client initialization.
+    #
+    # + clientConfig - Client configuration details
+    # + serviceUrl - Connector server URL
+    # + return - Returns error at failure of client initialization
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "localhost:9090/payloadV") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
```

Comment:
Shall we use a better name than `httpEp`

Reply:
This code is auto generated.